.github/workflows/scheduled.yaml

```diff
@@ -0,0 +1,49 @@
+# Tests in this workflow are run periodically.
+# They are generally very expensive and not on the critical path
+# for development
+# scheduled workflows only run in the default branch
+
+name: Scheduled Tests
+
+on:
+  schedule:
+    - cron: "42 3 * * 0"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
There is at least another almost-finalised workflow job that is expected to be triggered by on a schedule. Having a per-trigger workflow is consistent with what is done in quick.yaml and slow.yaml.

No objection to removing the concurrency clause

Reply:
> There is at least another almost-finalised workflow job that is expected to be triggered by on a schedule.

Anticipation of multiple scheduled jobs alone does not justify creating one workflow for all scheduled jobs. If possible, please, let's start simple instead of arguing about complex things that we do not need _right now_ (and that we definitely do not need to argue about right now).

> Having a per-trigger workflow is consistent with what is done in quick.yaml and slow.yaml.

That assertion is false because our quick.yaml and slow.yaml workflows are _not_ distinguished by their _triggers_. They actually _share_ certain triggers and their are organized/segregated based on very different principles!

 
> No objection to removing the concurrency clause

That removal is not enough, and spending time on discussing concurrency clause and other problems related to premature (at least) aggregation of future workflows is what this change request is trying to avoid. If we want minimal PRs, we should stop spending time on resisting PR minimization changes!

Reply:
I am going to resolve this outdated change request (because the file got renamed as requested) and open a new one to mark the corresponding missed updates (in the renamed file).